### PR TITLE
WEBAPP-351: dont hijack some cms paths

### PR DESCRIPTION
**Pull request by Schedulaar**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/routes/categories/containers/CategoriesPage.js [hunk 1]
```diff
@@ -53,7 +54,8 @@ export class CategoriesPage extends React.Component<PropsType> {
 
   redirectToPath = (path: string) => {
     const action = pathToAction(path, this.props.routesMap)
-    this.props.redirect(action)
+    setKind(action, 'push')
```
Comment (Schedulaar) [inline src/routes/categories/containers/CategoriesPage.js]: That router does suck though...

Comment (maxammann) [reply src/routes/categories/containers/CategoriesPage.js]: New change router task incoming? :P

File: src/modules/common/components/RemoteContent.js [hunk 2]
```diff
@@ -8,15 +8,17 @@ type PropsType = {
   dangerouslySetInnerHTML: {
     __html: string
   },
-  hosts: Array<string>,
+  hijackRegExp: RegExp,
   onInternLinkClick: string => void,
   centered: boolean
 }
 
+const HIJACK = /https?:\/\/(cms\.integreat-app\.de|web\.integreat-app\.de|integreat\.app)(?!\/[^/]*\/(wp-content|wp-admin|wp-json)\/.*).*/
```
Comment (maxammann) [inline src/modules/common/components/RemoteContent.js]: I'm a bit against using RegExps if this gets more complicated. For now it looks fine though.

Comment (Schedulaar) [reply src/modules/common/components/RemoteContent.js]: Hmm... rather a function, right? I will leave it for now.

**Pull request merged**